File: smartdevices-app/smartdevicesapp.model/src/main/java/de/vogler_engineering/smartdevicesapp/model/entities/ui/UiValueSpecification.java
/**
 * Copyright (c) Vogler Engineering GmbH. All rights reserved.
 * Licensed under the MIT License. See LICENSE.md in the project root for license information.
 */
package de.vogler_engineering.smartdevicesapp.model.entities.ui;

import de.vogler_engineering.smartdevicesapp.model.entities.value.ValueDataType;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class UiValueSpecification extends UiComponent {
    private ValueDataType DataType;
    private String DisplayType;
}

File: smartdevices-gateway/SmartDevicesGateway.Model/Ui/UiValueSpecification.cs
//
// Copyright (c) Vogler Engineering GmbH. All rights reserved.
// Licensed under the MIT License. See LICENSE.md in the project root for license information.
//
using System;
using System.Collections.Generic;
using System.Text;
using SmartDevicesGateway.Model.Config.ValueSpecifications;
using SmartDevicesGateway.Model.Values;

namespace SmartDevicesGateway.Model.Ui
{
    public class UiValueSpecification
    {
        public string Name { get; set; }

        public ValueDataSource DataSource { get; set; }

        public UiComponent Component { get; set; }

    }
}
